lib/active_model_serializers.rb

```diff
@@ -50,6 +51,7 @@ def silence_warnings
 
 require 'action_controller/serialization'
 ActiveSupport.on_load(:action_controller) do
+  ActiveSupport.run_load_hooks(:active_model_serializers, ActiveModelSerializers)
```

Comment:
the only thing that worries me here is that it was removed from AMS at one point